src/services/utilServices/MailClient.ts

```diff
@@ -28,16 +35,57 @@ class MailClient {
     }
 
     try {
-      await axios.post(endpoint, email, {
+      const sendMailResponse = await axios.post<MailData>(sendEndpoint, email, {
         headers: {
           Authorization: `Bearer ${this.POSTMAN_API_KEY}`,
         },
       })
+      this.verifyMail(sendMailResponse.data)
```

Comment:
Should we not await this for error handling purposes?

Reply:
Due to the delay being set to 1 minute, and that sendMail is used in `formsgSiteCreation.ts`, `formsgSiteLaunch.ts`, `UsersServices.ts` and `InfraServices.ts` for otps and other emails with await calls, I am not sure if awaiting for 1 minute will delay any important response from being sent. In terms of error handling, since the response for those errors would be to log them, I think there is no need to elevate the error. Wdyt?

Reply:
yea we should not be awaiting here as it may block other operations and the response from `sendMail` will be pending